lib/sqlite-store.js

```diff
@@ -1,433 +1,443 @@
-/*jslint node: true */
+/* jslint node: true */
 /* Copyright (c) 2012 Marius Ursache */
-"use strict";
+'use strict'
 
 // TODO: add some methods to create the test structure
 
-var sqlite  = require('sqlite3');
+var Sqlite = require('sqlite3')
 
-var _       = require('lodash');
-var uuid    = require('node-uuid');
-var error   = require('eraro')({package:'seneca-sqlite-store'});
+var _ = require('lodash')
+var Uuid = require('node-uuid')
+var error = require('eraro')({package: 'seneca-sqlite-store'})
 
-var name    = "sqlite-store";
+var name = 'sqlite-store'
 
-var MIN_WAIT = 16;
-//var MAX_WAIT = 65336;
+var MIN_WAIT = 16
+// var MAX_WAIT = 65336;
 
 module.exports = function (options) {
-  var desc;
-  var seneca = this;
-  var self = this;
+  var desc
+  var seneca = this
+  var self = this
 
   options = this.util.deepextend({
-    supefast:true
-  }, options);
+    supefast: true
+  }, options)
 
   var store = {
-    name:name,
+    name: name,
 
-    configure: function(spec, done){
-      self.spec = spec;
+    configure: function (spec, done) {
+      self.spec = spec
 
-      var conf = 'string' == typeof(spec) ? null : spec;
+      var conf = 'string' === typeof (spec) ? null : spec
 
-      if(!conf) {
-        conf = {};
-        //sqlite:///path/to/database;
-        var urlM = /^sqlite:\/\/(.*?)$/.exec(spec);
-        conf.database   = urlM[1];
+      if (!conf) {
+        conf = {}
+        // sqlite:///path/to/database;
+        var urlM = /^Sqlite:\/\/(.*?)$/.exec(spec)
```

Comment:
I don't think you need to change to capitalized here. it's an url format. See commented example